DocumentService/Word/WordDocumentGenerator.cs

```diff
@@ -244,7 +241,12 @@ private static void ReplaceImagePlaceholders(string inputFilePath, string output
 
                             using (var writer = new BinaryWriter(imagePart.GetStream()))
                             {
-                                writer.Write(File.ReadAllBytes(imagePlaceholders[docProperty.Name]));
+                                string imagePath = imagePlaceholders[docProperty.Name];
+
+                                using (WebClient webClient = new WebClient())
```

Comment:
`WebClient` has been deprecated and we need to use `HTTPClient`. This involves the methods to be asynchronous. We will take this up as an improvemnt. 
Add  a TODO for now.

Reply:
Added TODO for web client